src/legacy/core_plugins/kibana/public/discover/doc_table/components/table_row.js

```diff
@@ -20,15 +20,15 @@
 import _ from 'lodash';
 import $ from 'jquery';
 import rison from 'rison-node';
-import '../../doc_viewer';
-import '../../filters/uriescape';
-import '../../filters/short_dots';
-import { noWhiteSpace } from '../../../../core_plugins/kibana/common/utils/no_white_space';
+import 'plugins/kibana/doc_viewer';
+import 'ui/filters/uriescape';
+import 'ui/filters/short_dots';
+import { noWhiteSpace } from '../../../../common/utils/no_white_space';
```

Comment:
question: would the webpack alias work here, e.g. `plugins/kibana/common/utils/no_white_space`?

i know it works for stuff in the `public` directory but i don't know about `common`.

Reply:
I tried and it's not working with common :-(